app/views/service_providers/show.html.erb

```diff
@@ -13,7 +12,7 @@
 <p class="font-mono-xs margin-top-0" name="agency"><%= service_provider&.agency&.name %></p>
 
 <h2><label for="team">Team:</label></h2>
-<p class="font-mono-xs margin-top-0" name="agency"><%= service_provider.team %></p>
+<%= link_to "#{service_provider&.team}", team_path(service_provider&.team&.id), :class => 'usa-link' %>
```

Comment:
**suggestion:** I _think_ you might be able to make this work by just linking to the model. What do you think about wrapping this in a conditional to avoid all the nil checks? This should also allow us to leave the tests alone.

```suggestion
<% if service_provider.team.present? %>
  <%= link_to service_provider.team, service_provider.team, :class => 'usa-link' %>
  <% end %>
```

P.S. Rails magic is magical

Reply:
Also, `"#{service_provider&.team}"` the `#{}` is not needed since the entirety of the string is the thing inside the interpolation